src/parser/parser.yy

```diff
@@ -2508,6 +2512,24 @@ drop_snapshot_sentence
     }
     ;
 
+add_listener_sentence
+    : KW_ADD KW_LISTENER name_label host_list {
```

Comment:
Would it be better to replace name_label to KW_ELASTICSEARCH ?

Reply:
fixed